apps/api-key-service/src/models/apiKeys.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  index,
+  pgEnum,
+  pgTable,
+  timestamp,
+  uuid,
+  varchar,
+} from 'drizzle-orm/pg-core'
+
+export const apiKeyStateEnum = pgEnum('api_key_state', [
```

Comment:
We were planning on using strings instead of enums, we actually haven't granted the permission for enums in our script. Enums don't get stored in the database as a plain string, so if we ever imported our data from the db into another system for data science or audit purposes they would also need to import some mapping of strings to figure out what means what. In generally i think using just strings and enforcing with typechecks on the codelevel will be better in the long term

Reply:
got it - will replace in favor of strings typechecked at app level